sdk/monitor/azure-monitor-query/azure/monitor/query/_helpers.py

```diff
@@ -59,8 +61,10 @@ def construct_iso8601(start=None, end=None, duration=None):
         elif duration is not None:
```

Comment:
What's the behavior if user provides start, end and duration?

Reply:
we would take the start and end time - we have documented that all three should not be provided - on retrospect, let me warn here too

Reply:
In fact my question is if three are provided, we would silently ignore duration (current behavior) or give error? :)

Reply:
we now give error :)